modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -48,6 +48,36 @@
 }
 
 // Sidebar
+
+.has-control-menu .sidebar {
+	height: calc(100% - 56px);
+	top: 56px;
+}
+
+.sidebar {
+	height: 100%;
+	position: fixed;
+	right: -320px;
+	width: 320px;
+	z-index: 999; // Dropdowns are 1000
```

Comment:
Is that defined in a variable anywhere? ([liferay-frontend-guidelines discussion about `z-index`](https://github.com/liferay/liferay-frontend-guidelines/issues/104))

Reply:
we have [this one](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/variables/_globals-z-index.scss), I think we could try something like `z-index: $zindex-dropdown - 1;` if we want to keep it in sync with the dropdown

Reply:
Nice!